index.html

```diff
@@ -55,5 +59,236 @@ <h4 class="headline-sm-suport">LET'S CONNECT</h4>
+            </p>
+            <ul class="works-card-tags">
+              <li class="works-card-tag">html</li>
+              <li class="works-card-tag">Ruby on rails</li>
+              <li class="works-card-tag">css</li>
+              <li class="works-card-tag">javascript</li>
+            </ul>
+            <button>See Project</button>
+          </div>
+        </li>
+        <!-- Card 2 Ends -->
+      
+
+
+    <!-- About Section Starts Here -->
+    <section class="about">
+      <div class="headline-skill-container">
+        <div class="about-headline">
+          <h2 class="headline-heading heading-large">About Myself</h2>
+          <p class="headline-paragraph paragraph">
+            Hello I’m a software developer! I can help you build a product ,
+             feature or website. Look through some of my work and experience!
+             If you like what you see and have a project you need coded,
+              don’t hesitate to contact me.
+          </p>
+          <p class="headline-sm-suport">LET'S CONNECT</p>
+          <ul class="social-icons">
+            <li>
+              <a href="#"><i class="fa-brands fa-github"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-linkedin-in"></i></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-angellist"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-solid fa-m"></i></a>
+            </li>
+          </ul>
+          <button>Get my resume</button>
+        </div>
+        <ul class="about-skills-container">
+          <li class="about-skill-container">
+            <div class="about-skill-heading">
+              <h4 class="about-heading-small">Languages</h4>
+              <i class="fa-solid fa-angle-down"></i>
+            </div>
+            <div class="about-skill-body-container">
+              <div class="about-skill-body">
+                <img
+                  class="about-skill-body-img"
+                  src="./assets/Ellipse javascript 1.png"
+                  alt="js"
+                />
+                <h5 class="about-skill-text">Javascript</h5>
+              </div>
+              <div class="about-skill-body">
+                <img
+                  class="about-skill-body-img"
+                  src="./assets/Ellipse html 1.png"
+                  alt="html"
+                />
+                <h5 class="about-skill-text">HTML</h5>
+              </div>
+              <div class="about-skill-body">
+                <img
+                  class="about-skill-body-img"
+                  src="./assets/Ellipse css 1.png"
+                  alt="css"
+                />
+                <h5 class="about-skill-text">CSS</h5>
+              </div>
+            </div>
```

Comment:
- [ ] Kindly consider taking a ul at line 204 instead of div. then each language (JavaScript or HTML or CSS) will be taken as a li. that's how you will be following good semantics.